packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -61,7 +61,7 @@ import 'text_style.dart';
 ///  * [RichText], a widget for finer control of text rendering.
 ///  * [TextPainter], a class for painting [TextSpan] objects on a [Canvas].
```

Comment:
Add `ReactiveTextSpan` to the see also section here to make it more accessible/obvious.

Reply:
Also, there is still recognizer documentation in this doc.